src/edu/csus/ecs/pc2/ui/EditProblemSandboxPane.java

```diff
@@ -192,6 +182,40 @@ protected void initializeFields(Problem inProblem) {
         }
     }
     
+    /**
+     * Populate controls with problem's sandbox information
+     * 
+     * @param problem - the problem to use to populate the controls.  If null, uses defaults
+     */
+    private void populateGUI(Problem problem) {
+        
+        // If adding a problem, we have to determine a default sandbox type
+        if(problem == null) {
+            // If sandboxes are desired and supported, we make the internal sandbox the default
+            if(Problem.DEFAULT_SANDBOX_TYPE == SandboxType.PC2_INTERNAL_SANDBOX && isCheckSandboxSupport()) {
+                getUsePC2SandboxRadioButton().setSelected(true);
+            } else {
+                getUseNoSandboxRadioButton().setSelected(true);
+            }
+            getPC2SandboxOptionMemLimitTextbox().setText(Integer.toString(Problem.DEFAULT_MEMORY_LIMIT_MB));
+       } else {
+            //activate the appropriate sandbox selection button (the buttons are in a group so selecting one will deselect the others)
+            if (problem.getSandboxType() == SandboxType.NONE) {
+                getUseNoSandboxRadioButton().setSelected(true);
+                
+            } else if (problem.getSandboxType() == SandboxType.PC2_INTERNAL_SANDBOX) {
+                isCheckSandboxSupport();
+                getUsePC2SandboxRadioButton().setSelected(true);
```

Comment:
It bothers me greatly to see a call to a method that returns a boolean which is then ignored.  Shouldn't a return value of `false` here cause some kind of notification to the user?